Examples/Movies/__tests__/getImageSource-test.js

```diff
@@ -5,7 +5,8 @@
 
 jest.dontMock('../getImageSource');
 
-var getImageSource = require('../getImageSource');
+//see https://github.com/facebook/jest/issues/666
+const getImageSource = require('../getImageSource').default;
```

Comment:
as of latest jest, I think you can just do `import getImageSource from '../getImageSource';`